### PR TITLE
Match the latest UX spec #49

### PR DESCRIPTION
**Pull request by wisniewskit**

### CODE REVIEW

File: webextension/_locales/en_US/messages.json [hunk 1]
```diff
@@ -16,7 +16,10 @@
     "message": "Remove screenshot"
   },
   "buttonTakeScreenshot": {
-    "message": "Provide a screenshot"
+    "message": "Attach a Screenshot"
+  },
+  "buttonOK": {
+    "message": "Ok"
```
Comment (miketaylr) [inline webextension/_locales/en_US/messages.json]: Can you change this to OK per #51?

(Late breaking news...)

File: webextension/popup.html [hunk 2]
```diff
@@ -41,7 +46,11 @@ <h1></h1>
         </select>
         <textarea name="issueDescription" rows="1"></textarea>
         <button id="issueTakeScreenshot"></button>
-        <button id="issueRemoveScreenshot"></button>
+        <div id="issueRemoveScreenshot">
+          <p></p>
+          <img src="data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=" alt="" />
```
[the remark this answers is not in the document]
Comment (wisniewskit) [reply webextension/popup.html]: Yeah, I need to provide a `src` for the linter, so I'm just filling in a blank stand-in image. It's less overall code than adding/removing a tag via JS.

[the remark this answers is not in the document]
Comment (miketaylr) [reply webextension/popup.html]: Could we just have the image in the DOM as display:none, then add and remove a class `is-active` or something to show it? This feels a bit hacky (but is nice and clever :P)

[the remark this answers is not in the document]
Comment (wisniewskit) [reply webextension/popup.html]: Well, either the HTML linter has to be relaxed (to allow empty `src` attributes), or I'll have to remove the `img` tag from the HTML and add it dynamically with JS. I don't really have a preference either way, though.

[the remark this answers is not in the document]
Comment (miketaylr) [reply webextension/popup.html]: Why does the img need an empty `src`?

``

```
.remove-screenshot {
  display:none;
}

.remove-screenshot.is-active {
 display:block
}
```

(it's possible I'm missing something how this actually works!)

[the remark this answers is not in the document]
Comment (wisniewskit) [reply webextension/popup.html]: Oh, that might be fine too. But to be honest it also feels similarly hacky compared to using an empty image data URL, if foo.png wouldn't actually exist.

[the remark this answers is not in the document]
Comment (wisniewskit) [reply webextension/popup.html]: FWIW, I'll just use Mike's approach, and set "screenshot.jpg" as the src in the HTML, but also give it a style of `display:none`. The JS will then over-ride that display style as appropriate to show the image.

**Pull request merged**